cmd/crates/soroban-rpc-client/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
-name = "soroban-rpc"
+name = "soroban-rpc-client"
```

Comment:
Should we be renaming it to stellar-rpc-client given the rpc will eventually be losing the soroban name?

Reply:
Valid question !! Asking in `soroban-dev` to get feedback from product folks